src/containers/SideNav/SideNav.js

```diff
@@ -238,15 +239,21 @@ class SideNav extends Component {
             )}
           </SideNavMenu>
 
-          <SideNavMenuItem
-            element={NamespacesDropdown}
-            id="sidenav-namespace-dropdown"
-            selectedItem={{ id: namespace, text: namespace }}
-            showAllNamespaces
-            onChange={this.selectNamespace}
-          >
-            &nbsp;
-          </SideNavMenuItem>
+          {this.props.tenantNamespace ? (
+            <SideNavMenu defaultExpanded title="Namepsace">
```

Comment:
using a menu here seems a little odd but let's leave it for now since we'll be moving the dropdown out of the side nav soon anyway

typo:
```suggestion
            <SideNavMenu defaultExpanded title="Namespace">
```

Reply:
fixed